src/crimson/common/interruptible_future.h

```diff
@@ -1179,6 +1187,34 @@ struct interruptor
     return ::seastar::internal::when_all_succeed_impl(
 	futurize_invoke_if_func(std::forward<FutOrFuncs>(fut_or_funcs))...);
   }
+
+  template <typename Func,
+	    typename Result = futurize_t<std::invoke_result_t<Func>>>
+  static inline Result async(Func&& func) {
+    return seastar::async([func=std::forward<Func>(func),
+			   interrupt_condition=interrupt_cond<InterruptCond>]() mutable {
+      return non_futurized_call_with_interruption(
+	  interrupt_condition, std::forward<Func>(func));
+    });
+  }
+
+  static void yield() {
+    assert(interrupt_cond<InterruptCond>);
+    auto interruption_condition = interrupt_cond<InterruptCond>;
+    interrupt_cond<InterruptCond>.release();
+    seastar::thread::yield();
+    interrupt_cond<InterruptCond> = interruption_condition;
+  }
+
+  static void maybe_yield() {
```

Comment:
I don't think we use `yield()` nor `maybe_yield()` at the moment; I interpret their presence as a way to tell programmers these functions must be used instead of the vanilla ones.